src/plugins/data/public/ui/query_editor/_query_editor.scss

```diff
@@ -199,15 +202,17 @@
 .osdQuerEditor__singleLine {
   padding: $euiSizeS;
   background-color: $euiColorEmptyShade;
-  overflow: initial !important; // needed for suggestion window, otherwise will be hidden in child
+  overflow: visible !important; // needed for suggestion window
+  width: 100%; // Ensure it takes full width of its container
 
   .monaco-editor .view-overlays .current-line {
     border: none;
   }
 }
 
 .suggest-widget {
-  position: relative;
+  position: absolute;
+  z-index: 1000;
```

Comment:
z-index needs to be employed strategically due to all the other meaningful z-indexes. Have we verified that this plays nice with everything else and that `z-index: 1` is not good enough?

Reply:
good catch. I just switched to `z-index : 1`.